src/blueprints/tv2/timeline-object-factories/tv2-caspar-cg-timeline-object-factory.ts

```diff
@@ -18,8 +18,15 @@ import {
   Tv2VideoClipManifestData
 } from '../value-objects/tv2-action-manifest-data'
 
+enum CasparCgSlot {
+  FULL_GRAPHICS = '250_full',
+  IDENT = '650_ident',
+  LOWER_THIRD = '450_lowerThird',
+  UNKNOWN = ''
```

Comment:
It would be nice with a non-empty value for unknown, e.g. `UNKNOWN`. It would be more difficult to debug if we got an empty string in an error message.

Reply:
`UNKNOWN`'s content has been updated to `UNKNOWN`.